react-time-track/src/components/list-user-projects.js

```diff
@@ -0,0 +1,55 @@
+/** @jsx jsx */
+import React from "react";
+import { jsx } from "@emotion/core";
+import { navigate, Link } from "@reach/router";
+
+import { Card, Circle } from "./ui";
+import { Section } from "./helpers";
+import { listUserProjects } from "../services/project";
+
+import calculateProgress from "../utils/calculateProgress";
+import calculateRisk from "../utils/calculateRisk";
+
+const card = {
+  display: "flex",
+  justifyContent: "space-between",
+  alignItems: "center"
+};
+
+function ListUserProjects() {
+  const [userProjects, setUserProjects] = React.useState([]);
+
+  React.useEffect(() => {
+    listUserProjects()
+      .then(list => setUserProjects(list))
+      .catch(response => navigate("/login"));
```

Comment:
Hi, I comment that the context now has a method to log out, and I think you should only redirect to login when the error message is "Access denied", because you can receive another message which is not a error by authorization. You can check it the line 67 in the list-member-project file

Reply:
Thanks, I have made the change